app/controllers/buyers/bulk_base_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Buyers::BulkBaseController < FrontendController
+  before_action :authorize_bulk_operations
+
+  protected
+
+  def authorize_bulk_operations
+    authorize! :manage, scope
+  end
+
+  def permitted_params
+    params.premit(:selected)
```

Comment:
```suggestion
    params.permit(:selected)
```

If it does not fail then it looks like there is no test case for this :)